src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx

```diff
@@ -48,7 +48,7 @@ export const Timeline = ({ referendumId }: Props) => {
 
       {!isLoading &&
         timeline.map((status) => (
-          <div key={status.date.toLocaleString()} className="flex items-center justify-between">
+          <div key={`${status.status}-${status.date.toLocaleString()}`} className="flex items-center justify-between">
```

Comment:
I figured that 2 statuses were actually posted at the exact same time (the starting ones) resulting in an error in the console. Since it's a small one I added the fix in this PR.